tests/integration/ceph_multi_cluster_test.go

```diff
@@ -191,9 +171,7 @@ func (o MCTestOperations) Teardown() {
 
 func (o MCTestOperations) startCluster(namespace, store string) error {
 	logger.Infof("starting cluster %s", namespace)
-	useDevices := false
-	// do not use disks for this cluster, otherwise the 2 test clusters will each try to use the
-	// same disks.
+	useDevices := true
```

Comment:
Could we just remove the `useDevices` var now since your OSD PR merged? The CI will always use the devices. This could certainly be a separate PR.

Reply:
I think it's still nice to indicate the purpose of the bool, it's more meaningful than a "true" or "false" :).

Reply:
but isn't it always true now? Seems like we just don't need to pass this param anymore at all since we're always requiring devices.

Reply:
Done.